iot_api/user_api/models/DataCollector.py

```diff
@@ -140,6 +140,19 @@ def find_by_ids(cls, collector_ids):
         except Exception as e:
             LOG.error(e)
 
+            @classmethod
+
+    def find_by_ids_in_org(cls, collector_ids, organization_id):
+        try:
+            collectors = []
+            for collector_id in collector_ids:
+                collector = DataCollector.find_by_id(collector_id)
+                if collector.organization_id == organization_id
+                    collectors.append(collector)
+            return collectors
```

Comment:
I think is better to have only one method to find for entities, and use optional parameters to choose which filter criterias should be added to the query. See for example the method find_with in the Alert class.

For now we could leave the other methods as they are, but create a new one that have all possible filter criterias as optional parameters.